### PR TITLE
сhore: use upstream kurtosis package

### PR DESCRIPTION
**Pull request by klkvr**
Uses https://github.com/ethpandaops/optimism-package/pull/85 for now, once merged can be replaced with upstream package.

This also switches to using images from https://github.com/paradigmxyz/optimism cc @Rjected

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -34,21 +34,17 @@ jobs:
...
-        run: |
-            git clone https://github.com/klkvr/optimism-package
-            cd optimism-package
-            git switch odyssey
       - name: Build Odyssey
         run: docker buildx build . --load -t ghcr.io/ithacaxyz/odyssey:latest
       - name: Run enclave
         id: kurtosis
         run: |
+            git clone https://github.com/klkvr/optimism-package
```
Comment (jenpaff) [inline .github/workflows/e2e.yml:42]: why do we still need to use your optimism-package fork?

Comment (klkvr) [reply .github/workflows/e2e.yml:42]: once https://github.com/ethpandaops/optimism-package/pull/85 is merged there would be no need for it, can hold off this PR until it's merged

Comment (jenpaff) [reply .github/workflows/e2e.yml:42]: ah got it, no need to wait- we can just change it later was just curious

**Review by jenpaff — APPROVED** [review]
lgtm!

File: etc/kurtosis.yaml [hunk 2]
```diff
@@ -3,10 +3,25 @@ ethereum_package:
...
-    network_id: "41144114"
-    seconds_per_slot: 1
+  chains:
+    - participants:
+      - el_type: op-reth
+        el_image: ghcr.io/ithacaxyz/odyssey:latest
+        cl_image: ghcr.io/paradigmxyz/op-node:latest
+        el_extra_env_vars:
+          "EXP1_SK": "0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba"
+          "EXP1_WHITELIST": "0x0116686E2291dbd5e317F47faDBFb43B599786Ef"
```
Comment (onbjerg) [inline etc/kurtosis.yaml:13]: nit, think the current format is not normal in yaml

```suggestion
          EXP1_SK: "0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba"
          EXP1_WHITELIST: "0x0116686E2291dbd5e317F47faDBFb43B599786Ef"
```

File: etc/kurtosis.yaml [hunk 3]
```diff
@@ -3,10 +3,25 @@ ethereum_package:
...
+        el_extra_params:
+          - "--rollup.discovery.v4"
+      - el_type: op-reth
+        el_image: ghcr.io/ithacaxyz/odyssey:latest
+        cl_image: ghcr.io/paradigmxyz/op-node:latest
+        el_extra_env_vars:
+          "EXP1_SK": "0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba"
+          "EXP1_WHITELIST": "0x0116686E2291dbd5e317F47faDBFb43B599786Ef"
```
Comment (onbjerg) [inline etc/kurtosis.yaml:21]: same
```suggestion
          EXP1_SK: "0x8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba"
          EXP1_WHITELIST: "0x0116686E2291dbd5e317F47faDBFb43B599786Ef"
```

**Review by onbjerg — APPROVED** [review]

**Pull request merged**